warehouse/metrics_mesh/oso_metrics/active_developers.sql

```diff
@@ -0,0 +1,14 @@
+select @metrics_sample_date(events.bucket_day) as metrics_sample_date,
+  events.event_source,
+  events.to_artifact_id as project_id,
+  '' as from_artifact_id,
+  @metric_name() as metric,
+  COUNT(distinct events.from_artifact_id) as amount
+from metrics.events_daily_to_artifact as events
+where events.event_type = 'COMMIT_CODE'
+  and events.bucket_day BETWEEN @metrics_start(DATE) AND @metrics_end(DATE)
```

Comment:
```suggestion
  and events.bucket_day BETWEEN @metrics_start('DATE') AND @metrics_end('DATE')
```

Turns out I made an error here with my initial docs of this. It _should_ be a string input here. Eventually this will be used as a way to figure out the type casting and the macro needs this value as a string.